### ISSUE TITLE
[CAEP] Assurance Level Change: add new claim for namespace

### ISSUE THREAD

**Issue by timcappalli** [openid/sharedsignals#24]
We need a new claim that defines the namespace used for the assurance levels. Claim name TBD (example used is `namespace`).

```json
{
    "namespace": "",
    "previous_level": "",
    "current_level": ""
}
```

**Comment by tulshi** [comment]
Please review pull request: https://github.com/openid/sse/pull/31

**Comment by timcappalli** [comment]
2023-03-21 closing as this was addressed in PR #31
